skimage/feature/blob.py

```diff
@@ -337,9 +341,12 @@ def blob_dog(image, min_sigma=1, max_sigma=50, sigma_ratio=1.6, threshold=2.0,
     gaussian_images = [gaussian_filter(image, s) for s in sigma_list]
 
     # computing difference between two successive Gaussian blurred images
-    # multiplying with average standard deviation provides scale invariance
-    dog_images = [(gaussian_images[i] - gaussian_images[i + 1])
-                  * np.mean(sigma_list[i]) for i in range(k)]
+    # need to divide by (sigma_ratio-1) to obtain an approximation of the
+    # scale invariant Laplacian of the Gaussian operator
+    dog_images = [
+        (gaussian_images[i] - gaussian_images[i + 1]) / (sigma_ratio - 1)
+        for i in range(k)
+    ]
```

Comment:
```suggestion
    dog_images = [
        (gaussian_images[i] - gaussian_images[i + 1]) for i in range(k)
    ]
```

Reply:
I think we can remove the division by `sigma_ratio - 1` since it is only the relative value across scales that is used to find blobs